### PR TITLE
feat(docker): add Docker support

### PR DESCRIPTION
**Pull request by seanmonstar**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: '2'
```
Comment (jbuck) [inline docker-compose.yml]: I'd agree with your/Ryans assertion that we can delete for this now.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:trusty
```
Comment (vladikoff) [inline Dockerfile]: 👍

**Comment by seanmonstar** [comment]
@jbuck ok, updated. Added a circle.yml, and added integration. I'm noticing that the build is failing cause it's not published to the docker library. Is that required to run the tests? I'd think that running this on circle would use the code in the repo, not something already published...

File: Dockerfile-test [hunk 3]
```diff
@@ -0,0 +1,2 @@
+FROM fxa-custom-server:build
```
Comment (jbuck) [inline Dockerfile-test]: The reason it's trying to pull from docker hub is the missing `s` here, should be `fxa-customs-server`

**Comment by seanmonstar** [comment]
its working!!

File: Dockerfile-build [hunk 4]
```diff
@@ -0,0 +1,14 @@
+FROM mhart/alpine-node:4.7.2
```
Comment (jbuck) [inline Dockerfile-build]: Lets update this to 4.8.0 since that's out now

**Comment by seanmonstar** [comment]
Updated the alpine-node version to 4.8.0.

**Pull request merged**